lib/ansible/plugins/inventory/docker_machine.py

```diff
@@ -0,0 +1,175 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019, Ximon Eighteen <ximon.eighteen@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: docker_machine
+    plugin_type: inventory
+    short_description: Docker machine inventory source
+    requirements:
+        - Docker Machine
+    extends_documentation_fragment:
+        - inventory_cache
+        - constructed
+    description:
+        - Get inventory hosts from Docker Machine.
+        - Uses a YAML configuration file that ends with docker_machine.(yml|yaml).
+        - The plugin returns an 'all' group of nodes and one group per driver (e.g. digitalocean).
```

Comment:
```suggestion
        - The plugin returns an C(all) group of nodes and one group per driver (e.g. digitalocean).
```
([more info on formatting](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation))